ui/src/components/Nav/Desktop/index.tsx

```diff
@@ -39,27 +39,38 @@ const Desktop: React.FC<Props> = (props): React.ReactElement => (
                                     as="ul"
                                     className="divide-gray-100 absolute right-0 mt-2 w-56 origin-top-right divide-y divide-grey-200 divide-opacity-40 rounded-md border-2 border-transparent bg-white-50 shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none dark:divide-teal-600 dark:border-teal-500 dark:bg-grey-800"
                                 >
-                                    {item.subItems.map((subItem, index) => (
-                                        <li
-                                            key={index}
-                                            className="py-2 px-3 text-teal-600 transition-colors duration-500 dark:text-white-50"
-                                        >
-                                            <HeadlessUI.Menu.Item>
-                                                <div>
-                                                    {subItem?.label && subItem.value ? (
-                                                        <Components.Link
-                                                            href={subItem.value}
-                                                            className="block w-full rounded border-transparent outline-0 focus:ring-2 focus:ring-yellow-400"
+                                    {item.subItems.map((subItem, index) => {
+                                        return subItem?.label && subItem.value ? (
+                                            <li className="py-2 px-3 text-teal-600 transition-colors duration-500 dark:text-white-50">
+                                                <HeadlessUI.Menu.Item as="a" key={index} href={subItem.value}>
```

Comment:
as="a" makes the new page become blank for a couple of seconds while navigating to that route. You need to replace that "a" with the Link component from "next/link".

Reply:
The trouble here is when changed to Component.Link, the enter key press no longer works. If I remove Menu.Item and then I lose access to the "active" state which HeadlessUI seems to rely on for selection highlighting? Next Component.Link inside Menu.Item will keep the highlighting, but lose ability to use the keyboard to navigate.

Reply:
Strange, okay I did mention on Friday in standup about a spike ticket when you weren't around, about potentially other solutions, developing our own dropdown, other libraries etc, since HeadlessUI does not rely on tab focus anyway. 

Can't catch up today or tomorrow due being in Bristol, won't be around until Friday.

Reply:
Ok, no worries. I can do the changes and add a new commit on this PR. HeadlessUI works fine for this, I don't think we need other solutions.